CHANGES.md

```diff
@@ -14,6 +14,7 @@ Release Notes.
 - Use startup probe option for first initialization of application
 - Allow setting env for UI deployment.
 - Add Istio ServiceEntry permissions.
+- Integrate BanyanDB as storage solution
```

Comment:
4.5.0 is released, please don't change the changelog here, add a new section for 4.6.0